calamari_ocr/ocr/evaluator.py

```diff
@@ -66,6 +66,7 @@ def run(self, _sentinel=None, gt_dataset=None, pred_dataset=None, processes=1, p
 
         if self.preloaded_gt:
             gt_data = self.preloaded_gt
+            print("GT PRELOADED")
```

Comment:
Remove print. Seems to be a debug output.

Reply:
done